src/Files.App/Views/Properties/HashesPage.xaml.cs

```diff
@@ -46,6 +48,54 @@ private void ToggleMenuFlyoutItem_Click(object sender, RoutedEventArgs e)
 			_cancel = true;
 		}
 
+		private void HashInputTextBox_TextChanged(object sender, TextChangedEventArgs e)
```

Comment:
We can remove the event handler and directly bind the text to the view model. As the property updates, we can compare the hashes and update the info bar's visibility and message accordingly.